snap/local/register-device.sh

```diff
@@ -7,6 +7,19 @@ if [ ! -f "$CONFIGURATION_FILE_PATH" ]; then
     exit 1
 fi
 
-$SNAP/bin/cos-registration-agent --shared-data-path $SNAP_COMMON/rob-cos-shared-data --grafana-dashboards $SNAP_COMMON/configuration/grafana_dashboards --foxglove-studio-dashboards $SNAP_COMMON/configuration/foxglove_layouts setup -c $CONFIGURATION_FILE_PATH
+REGISTRATION_CMD="$SNAP/bin/cos-registration-agent --shared-data-path $SNAP_COMMON/rob-cos-shared-data"
+
+# Check if grafan_dashboards directory exists in configuration
+if [ -d "$SNAP_COMMON/configuration/grafana_dashboards" ]; then
+    REGISTRATION_CMD="$REGISTRATION_CMD --grafana-dashboards $SNAP_COMMON/configuration/grafana_dashboards"
+fi
+
+# Check if foxglove_layouts directory exists in configuration
+if [ -d "$SNAP_COMMON/configuration/foxglove_layouts" ]; then
+    REGISTRATION_CMD="$REGISTRATION_CMD --foxglove-studio-dashboards $SNAP_COMMON/configuration/foxglove_layouts"
+fi
+
+REGISTRATION_CMD="$REGISTRATION_CMD setup -c $CONFIGURATION_FILE_PATH"
```

Comment:
Is this a positional arg or could it be merged with the initial `REGISTRATION_CMD` declaration ? (e.g. append `--grafana-dashboards foo` after `setup -c bar`).

Reply:
it's a positional argument, so it has to be at the end